app.js

```diff
@@ -58,7 +52,68 @@ var itemObjects = [
 ];
 
 var itemsObjectsWorking;
-var itemsChosen = 0;
+var itemsChosen = window.localStorage.itemsChosen || 0;
+var itemsDisplayed = window.localStorage.itemsDisplayed || 0;
+
+function onLoadValues(){
+  updateItemObjects();
+  if(itemsChosen > 24){
+    var surveyButtonLocation = document.getElementById('startSurvey');
+    surveyButtonLocation.setAttribute('style', 'display:none');
```

Comment:
Best to use the [`style` property](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/style) instead of setting an attribute.